Q&A about ChrisM (#87)
tim-one/tim-one.github.io: psf/ban_qa.md

---
Brief replies to questions from a variety of sources, most recent first.

**2024-10-18 Fri 12:45a CDT**

**Q:** Se Ethan wasn't your friend - what about Chris McDonough?

**A:** Chris surprised me too! Chris was an old "work friend", back from when Zope Corp hired PythonLabs (2000). Due to odd working conditions (cities about 60 miles apart, usually working remotely, and carpooling when Guido, Fred Drake, and I did go to headquarters), I almost never saw my Zope coworkers socially outside of work. Relations were friendly among all, and Chris and I hit it off well, but no "real friendships" could grow on such arid land. When I left Zope Corp, we all fell out of touch at once.

> So it took me by surprise when Chris emailed to say he had written a blog post, and asked me to rewiew it. I was astonished at the clarity of thought and expression - I had no idea how talented an observer, thinker, and writer Chris was. I had some disagreements, but too minor to push. The influence of my review was limited to minor changes in the sections about Python's early days. For the rest, Chris speaks for Chris, and his voice is authentic. Preserving that authenticity was far more important to my eyes than risk losing it to "written by committee" quibbles over, e.g., guesses about what of some the charges against me were "really" about.

Author: ```suggestion
So it took me by surprise when Chris emailed to say he had written a blog post, and asked me to review it. I was astonished at the clarity of thought and expression - I had no idea how talented an observer, thinker, and writer Chris was. I had some disagreements, but too minor to push. The influence of my review was limited to minor changes in the sections about Python's early days. For the rest, Chris speaks for Chris, and his voice is authentic. Preserving that authenticity was far more important to my eyes than risk losing it to "written by committee" quibbles over, e.g., guesses about what of some the charges against me were "really" about.
```

> I loved the post mostly for its love letters to Python and its community. Which, like it or not, I did play foundational roles in creating. As I've mentioned elsewhere in my blog, the list of charges is so contrived they're not worth the bother of "refuting". Chris's analysis of those was the least interesting part to me, but Chtis put real effort into that too, and he deserved better than to have it offhandedly dismissed as "analysis". He put approximately infinitely more effort into analyzing the evidence than the ban announcement put into presenting a coherent case.

Author: ```suggestion
I loved the post mostly for its love letters to Python and its community. Which, like it or not, I did play foundational roles in creating. As I've mentioned elsewhere in my blog, the list of charges is so contrived they're not worth the bother of "refuting". Chris's analysis of those was the least interesting part to me, but Chris put real effort into that too, and he deserved better than to have it offhandedly dismissed as "analysis". He put approximately infinitely more effort into analyzing the evidence than the ban announcement put into presenting a coherent case.
```

> I loved the post mostly for its love letters to Python and its community. Which, like it or not, I did play foundational roles in creating. As I've mentioned elsewhere in my blog, the list of charges is so contrived they're not worth the bother of "refuting". Chris's analysis of those was the least interesting part to me, but Chtis put real effort into that too, and he deserved better than to have it off-handedly dismissed as "analysis". He put approximately infinitely more effort into analyzing the evidence than the ban announcement put into presenting a coherent case.

Author: ```suggestion
I loved the post mostly for its love letters to Python and its community. Which, like it or not, I did play foundational roles in creating. As I've mentioned elsewhere in my blog, the list of charges is so contrived they're not worth the bother of "refuting". Chris's analysis of those was the least interesting part to me, but Chtis put real effort into that too, and he deserved better than to have it offhandedly dismissed as "analysis". He put approximately infinitely more effort into analyzing the evidence than the ban announcement put into presenting a coherent case.
```
